src/modules/endpoint/reducers/pushCategory.js

```diff
@@ -52,7 +52,8 @@ const pushCategory = (state: CityContentStateType, action: PushCategoryActionTyp
         allAvailableLanguages: getAllAvailableLanguages(root, language, city, languages)
       }
     },
-    resourceCache: {...state.resourceCache, ...resourceCache}
+    resourceCache: {...state.resourceCache, ...resourceCache},
+    categoriesMap
```

Comment:
Hmm... not sure if this is the right spot to add the categoriesMap to the store... (Because we also have pushEvent and maybe multiple more)... I think I'd rather like to have a separate action for that, which is only called when opening the SearchModal which stores that categoriesMap in sth like `cityContent.searchRoute.categoriesMap` to have some encapsulated space just for the searchRoute 🤔 We could also clean this again once the route has disconnected... This would also follow the prior structure of the state a little bit better...

But you can decide :D

Reply:
We could move the data to a searchRoute prop, I have absolutely no problem with that. But adding a separate action adds unnecessary complexity imo. An additional saga/reducer would be needed. If at some point also events could be searched, I'd agree with you though. What do you think?

Reply:
@Schedulaar

Reply:
Well, I still think, this would be the better and more sound approach. I'd go for it, but I approved  this PR, so you could also merge it now :)

Reply:
Alright, I'll just gonna add a note to the corresponding task.